spec/components/schemas.yaml

```diff
@@ -644,11 +626,11 @@ components:
         bbox:
           $ref: "./schemas.yaml#/components/schemas/bbox"
         geometry:
+          $ref: "./schemas.yaml#/components/schemas/itemGeometry"
           # we could use the 'original' schema, but it doesn't
           # contain a useful example, hence we use our own
-          # schema with appropriate example
-          # $ref: https://geojson.org/schema/Geometry.json
-          $ref: "./schemas.yaml#/components/schemas/geometryGeoJSON"
+          # schema with appropriate example. Original schemas
+          # were defined here: https://geojson.org/schema/Geometry.json
         properties:
           $ref: "./schemas.yaml#/components/schemas/itemProperties"
           readOnly: true
```

Comment:
Not related to this PR but could be corrected here. The reference are local in the file therefore the prefix `./schemas.yaml` could be removed here

Reply:
removed those prefixes in the whole `schemas.yaml`